guides/common/modules/proc_configuring-the-project-mcp-server.adoc

```diff
@@ -0,0 +1,35 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="configuring-the-{project-context}-mcp-server"]
+= Configuring the {Project} MCP server
+
+Deploy the {Project} MCP server as a container and make it accessible to your MCP client.
+Perform these steps on the same system where your MCP client will run.
+
+.Procedure
+. Optional: Pull the latest version of the MCP container image from the registry:
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+$ podman pull {mcp-server-image}
```

Comment:
On Satellite, it may (or may not) be necessary to log into the registry before pulling the image. cc @ehelms